test/deal-folder/post-document-in-deal-folder.api-test.ts

```diff
@@ -114,15 +115,15 @@ describe('postDocumentInDealFolder', () => {
           { ...uploadFileInDealFolderRequest[0], documentType },
         ];
 
-        const updateFileInfoRequest = {
-          Title: documentTitle,
-          Document_x0020_Status: DOCUMENT_X0020_STATUS,
-          [ENVIRONMENT_VARIABLES.SHAREPOINT_ESTORE_DOCUMENT_TYPE_ID_FIELD_NAME]: documentTypeId,
+        const modifiedUpdateFileInfoRequest = {
+          ...updateFileInfoRequest,
         };
+        modifiedUpdateFileInfoRequest.fields.Title = documentTitle;
+        modifiedUpdateFileInfoRequest.fields[ENVIRONMENT_VARIABLES.SHAREPOINT_ESTORE_DOCUMENT_TYPE_ID_FIELD_NAME] = documentTypeId;
```

Comment:
Personally I prefer writing this as:
```
const modifiedUpdateFileInfoRequest = {
  ...updateFileInfoRequest,
  fields: {
    ...updateFileInfoRequest.fields,
    Title: documentTitle,
    [ENVIRONMENT_VARIABLES.SHAREPOINT_ESTORE_DOCUMENT_TYPE_ID_FIELD_NAME]: documentTypeId, 
  },
};
```
I think it's a bit easier to read because it's declarative, and I like that it means you don't mutate `modifiedUpdateFileInfoRequest` at all. It's up to you if you want to change it :)

Reply:
I also prefer this, have made this change